pkg/api/v1/types.go

```diff
@@ -892,6 +892,9 @@ type PodSpec struct {
 	// used must be specified.
 	// Optional: Default to false.
 	HostNetwork bool `json:"hostNetwork,omitempty" description:"host networking requested for this pod"`
+	// Run the container using the host ipc mode.
+	// Optional: Default to false.
+	HostIPC bool `json:"hostIPC,omitempty" description:"host ipc requested for this pod"`
```

Comment:
The description should be at least as explanatory as the comment, if not more so. Users only read the content of the description tags.

Hopefully we'll be able to unify the two soon -- #10933.